### ISSUE TITLE
Invitation links to the same user are not generated correctly after the second time.

### ISSUE THREAD

**Issue by kurehajime** [langgenius/dify#5391]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.6.11

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

1. invite users. But forget to save the invitation link. 
2. invite the user again. The invitation link is not generated properly and the URL of the login page is generated.
3. delete the user
4. invite the user again, but the invitation link is not created correctly

<img width="426" alt="名称未設定" src="https://github.com/langgenius/dify/assets/4569916/03090377-6f7b-48fc-84e5-d0f2fc3aea33">

Invitation links to other users are correctly generated.
This problem is serious in environments where a mail server is not provided.

### ✔️ Expected Behavior

- Invitation links can be generated as many times as required.
- Invitation links are generated correctly when re-inviting a deleted user.

### ❌ Actual Behavior

- Invitation links are not generated correctly the second time or later
- Invitation links are not generated correctly for users who have been deleted once.

**Comment by kurehajime** [comment]
Thank you dosubot 👍 
Deleting the data in the `accounts` table correctly created the invitation link.

I have found a workaround, but it would be a better product if there was also a way to resolve this from the GUI.

**Comment by yaohongfenglove** [comment]
Has this problem not been solved yet? The issue with version 0.6.13 still recurs

**Comment by prd-tien-ngo** [comment]
Hi there!

I am deploying 0.6.14 to kubernetes cluster. Everythings seems good exception send-mail function.

I am using `resend` mail type by setting below variables for api pod

```
    - name: MAIL_TYPE
      value: "resend"
    - name: RESEND_API_KEY
      value: "**********"
    - name: RESEND_API_URL
      value: "https://api.resend.com"
```


Whenever I creating a new user, no invitation link appears

<img width="959" alt="image" src="https://github.com/user-attachments/assets/740f4c34-de87-4f41-9433-656933a6832e">

and the `invite-email` call is timeout with http 504 code

<img width="817" alt="image" src="https://github.com/user-attachments/assets/b1ae6002-daf5-4c17-b944-465e9e87fc2f">


And the following logs appeared in the api pod (I enabled debug logging)

```
2024-07-17 06:49:30,598.598 DEBUG [Dummy-13] [functional.py:335] -
def document_indexing_sync_task(dataset_id, document_id):
    return 1

2024-07-17 06:49:30,600.600 DEBUG [Dummy-13] [functional.py:335] -
def clean_notion_document_task(document_ids, dataset_id):
    return 1

2024-07-17 06:49:30,600.600 DEBUG [Dummy-13] [functional.py:335] -
def batch_import_annotations_task(job_id, content_list, app_id, tenant_id, user_id):
    return 1

2024-07-17 06:49:30,601.601 DEBUG [Dummy-13] [functional.py:335] -
def add_document_to_index_task(dataset_document_id):
    return 1

2024-07-17 06:49:30,601.601 DEBUG [Dummy-13] [functional.py:335] -
def clean_dataset_task(dataset_id, tenant_id, indexing_technique, index_struct, collection_binding_id, doc_form):
    return 1

2024-07-17 06:49:30,601.601 DEBUG [Dummy-13] [functional.py:335] -
def retry_document_indexing_task(dataset_id, document_ids):
    return 1

2024-07-17 06:49:30,601.601 DEBUG [Dummy-13] [functional.py:335] -
def document_indexing_task(dataset_id, document_ids):
    return 1

2024-07-17 06:49:30,602.602 DEBUG [Dummy-13] [functional.py:335] -
def accumulate(self, *args, **kwargs):
    return 1

2024-07-17 06:49:30,602.602 DEBUG [Dummy-13] [functional.py:335] -
def process_trace_tasks(tasks_data):
    return 1

2024-07-17 06:49:30,602.602 DEBUG [Dummy-13] [functional.py:335] -
def update_annotation_to_index_task(annotation_id, question, tenant_id, app_id, collection_binding_id):
    return 1

2024-07-17 06:49:30,603.603 DEBUG [Dummy-13] [functional.py:335] -
def unlock_chord(self, group_id, callback, interval=0, max_retries=1, result=2, Result=3, GroupResult=4, result_from_tuple=5, **kwargs):
    return 1

2024-07-17 06:49:30,603.603 DEBUG [Dummy-13] [functional.py:335] -
def xmap(task, it):
    return 1

2024-07-17 06:49:30,603.603 DEBUG [Dummy-13] [functional.py:335] -
def sync_website_document_indexing_task(dataset_id, document_id):
    return 1

2024-07-17 06:49:30,603.603 DEBUG [Dummy-13] [functional.py:335] -
def delete_annotation_index_task(annotation_id, app_id, tenant_id, collection_binding_id):
    return 1

2024-07-17 06:49:30,604.604 DEBUG [Dummy-13] [functional.py:335] -
def remove_document_from_index_task(document_id):
    return 1

2024-07-17 06:49:30,604.604 DEBUG [Dummy-13] [functional.py:335] -
def deal_dataset_vector_index_task(dataset_id, action):
    return 1

2024-07-17 06:49:30,604.604 DEBUG [Dummy-13] [functional.py:335] -
def xstarmap(task, it):
    return 1

2024-07-17 06:49:30,604.604 DEBUG [Dummy-13] [functional.py:335] -
def document_indexing_update_task(dataset_id, document_id):
    return 1

2024-07-17 06:49:30,605.605 DEBUG [Dummy-13] [functional.py:335] -
def batch_create_segment_to_index_task(job_id, content, dataset_id, document_id, tenant_id, user_id):
    return 1

2024-07-17 06:49:30,605.605 DEBUG [Dummy-13] [functional.py:335] -
def chunks(task, it, n):
    return 1

2024-07-17 06:49:30,605.605 DEBUG [Dummy-13] [functional.py:335] -
def clean_document_task(document_id, dataset_id, doc_form, file_id):
    return 1

2024-07-17 06:49:30,605.605 DEBUG [Dummy-13] [functional.py:335] -
def send_invite_member_mail_task(language, to, token, inviter_name, workspace_name):
    return 1

2024-07-17 06:49:30,605.605 DEBUG [Dummy-13] [functional.py:335] -
def group(self, tasks, result, group_id, partial_args, add_to_parent=0):
    return 1

2024-07-17 06:49:30,606.606 DEBUG [Dummy-13] [functional.py:335] -
def delete_segment_from_index_task(segment_id, index_node_id, dataset_id, document_id):
    return 1

2024-07-17 06:49:30,606.606 DEBUG [Dummy-13] [functional.py:335] -
def chain(*args, **kwargs):
    return 1

2024-07-17 06:49:30,606.606 DEBUG [Dummy-13] [functional.py:335] -
def disable_annotation_reply_task(job_id, app_id, tenant_id):
    return 1

2024-07-17 06:49:30,606.606 DEBUG [Dummy-13] [functional.py:335] -
def duplicate_document_indexing_task(dataset_id, document_ids):
    return 1

2024-07-17 06:49:30,606.606 DEBUG [Dummy-13] [functional.py:335] -
def chord(self, header, body, partial_args=0, interval=1, countdown=2, max_retries=3, eager=4, **kwargs):
    return 1

2024-07-17 06:49:30,606.606 DEBUG [Dummy-13] [functional.py:335] -
def enable_segment_to_index_task(segment_id):
    return 1

2024-07-17 06:49:30,607.607 DEBUG [Dummy-13] [functional.py:335] -
def backend_cleanup():
    return 1

2024-07-17 06:49:30,607.607 DEBUG [Dummy-13] [functional.py:335] -
def add_annotation_to_index_task(annotation_id, question, tenant_id, app_id, collection_binding_id):
    return 1

2024-07-17 06:49:30,607.607 DEBUG [Dummy-13] [functional.py:335] -
def send_reset_password_mail_task(language, to, token):
    return 1

2024-07-17 06:49:30,607.607 DEBUG [Dummy-13] [functional.py:335] -
def disable_segment_from_index_task(segment_id):
    return 1

2024-07-17 06:49:30,607.607 DEBUG [Dummy-13] [functional.py:335] -
def enable_annotation_reply_task(job_id, app_id, user_id, tenant_id, score_threshold, embedding_provider_name, embedding_model_name):
    return 1

2024-07-17 06:49:30,608.608 DEBUG [Dummy-13] [functional.py:335] -
def recover_document_indexing_task(dataset_id, document_id):
    return 1

2024-07-17 06:49:30,608.608 DEBUG [Dummy-13] [functional.py:335] -
def remove_app_and_related_data_task(self, app_id):
    return 1
```

Could anyone advise how to fix the issue? is there a way that I can bypass the invitation email-sending function? Only invitation link is good enough for my case.

Thanks!
